docs/tutorials/add-caching-to-pg-with-redis.rst

```diff
@@ -0,0 +1,159 @@
+Tutorial: Add caching to your PostgreSQL® app using Redis®*
+===========================================================
+
+
+Learning objectives
+-------------------
+
+* Creating a simple Python web application, that accesses data in a PostgreSQL database
```

Comment:
Are we specifically doing a cache to improve read performance, or are we addressing write performance at any point in this tutorial?

Reply:
Since read caching is what I have worked with, and it's easy to explain, that's all I'd expected to address in the tutorial.

Write caching (and not losing the data if the cache goes away?) isn't something I really know about.

(But I'm more than willing to defer to more knowledge!)